client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -0,0 +1,390 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import isEmpty from 'lodash/lang/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'analytics';
+import acceptDialog from 'lib/accept';
+import DisconnectJetpackDialog from 'my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog';
+import PluginItem from 'my-sites/plugins/plugin-item/plugin-item';
+import PluginsActions from 'lib/plugins/actions';
+import PluginsListHeader from 'my-sites/plugins/plugin-list-header';
+import PluginsLog from 'lib/plugins/log-store';
+import PluginNotices from 'lib/plugins/notices';
+import SectionHeader from 'components/section-header';
+
+export default React.createClass( {
+	displayName: 'PluginsList',
+
+	mixins: [ PluginNotices ],
+
+	propTypes: {
+		plugins: PropTypes.array,
```

Comment:
I like this idea. I will just add the shape that is being used in this component.